its/src/test/java/com/sonar/it/scanner/msbuild/TestUtils.java

```diff
@@ -70,17 +75,35 @@ private static MavenLocation mavenLocation(String scannerVersion) {
   public static ScannerForMSBuild newScanner(Orchestrator orchestrator, Path projectDir) {
     String scannerVersion = getScannerVersion(orchestrator);
 
+    Location scannerLocation;
     if (scannerVersion != null) {
       LOG.info("Using Scanner for MSBuild " + scannerVersion);
-      return ScannerForMSBuild.create(projectDir.toFile())
-        .setScannerLocation(mavenLocation(scannerVersion));
-    } else {
-      // run locally
-      LOG.info("Using Scanner for MSBuild from the local build");
-      Path scannerZip = Paths.get("../DeploymentArtifacts/BuildAgentPayload/Release/sonarscanner-msbuild-net46.zip");
-      return ScannerForMSBuild.create(projectDir.toFile())
-        .setScannerLocation(FileLocation.of(scannerZip.toFile()));
+      scannerLocation = mavenLocation(scannerVersion);
     }
+    else {
+      String scannerLocationEnv = System.getenv("SCANNER_LOCATION");
```

Comment:
Added SCANNER_LOCATION environment variable as a way to specify the location of the scanner - easy to specify in a VSTS build to point at the build artifacts to use.